src/util.cc

```diff
@@ -423,21 +426,132 @@ const char* SpellcheckString(const char* text, ...) {
 }
 
 #ifdef _WIN32
+std::wstring Utf8ToWide(const StringPiece& u8_string) {
+  if (u8_string.len_ == 0)
+    return std::wstring();
```

Comment:
@jlonnberg 

Does MultiByteToWideChar break if you pass it a string of zero length?

While you're saving that function call in the error-situation, you're also adding extra work for the "happy-path" for every call.

I'm not sure it's worth making this check, unless MultiByteToWideChar breaks on a zero length input.

Reply:
There is no way around this check. The method calculating the size of the new string will return zero when it fails, i.e., the same value as when converting zero bytes.

Reply:
Alright.